src/pages/contact.js

```diff
@@ -0,0 +1,185 @@
+/** @jsx jsx */
+import { useCallback, useState } from 'react'
+import { Redirect } from '@reach/router'
+import {
+  Button,
+  Text,
+  Box,
+  Label,
+  Input,
+  Select,
+  Textarea,
+  jsx,
+} from 'theme-ui'
+
+import Layout from '../components/layout'
+import PageHeader from '../components/pageheader'
+import Container from '../components/container'
+
+import getFirebase from '../firebase'
+
+export default function login() {
+  const firebaseApp = getFirebase()
+
+  const [error, setError] = useState(null)
+  const [success, setSuccess] = useState(null)
+
+  const [name, setName] = useState('')
+  const handleSetName = useCallback(e => setName(e.target.value), [name])
+
+  const [email, setEmail] = useState('')
+  const handleSetEmail = useCallback(e => setEmail(e.target.value), [email])
+
+  const [type, setType] = useState('Student')
+  const handleSetType = useCallback(e => setType(e.target.value), [type])
```

Comment:
Unnecessary dependency, please remove.
```suggestion
  const handleSetType = useCallback(e => setType(e.target.value), [])
```